src/KeyVault/KeyVault/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Added Secret URI Parameter to Key Vault Secret Cmdlets [#23053]
+* Upgraded Azure.Core to 1.44.1.
```

Comment:
```suggestion
```
Azure core is not updated in this PR

Reply:
This was a merge conflict from the main branch